bom/application/pom.xml

```diff
@@ -159,6 +159,7 @@
         <flyway.version>7.14.0</flyway.version>
         <yasson.version>1.0.9</yasson.version>
         <liquibase.version>4.4.3</liquibase.version>
+        <liquibase-mongodb.version>4.4.3</liquibase-mongodb.version>
```

Comment:
A bit late to the party but is there a reason to have a separate version? Aren't they part of the same build?

Asking because it might be a pain for Dependabot.

Reply:
I'll check, when I first start working on it month ago they had totally different versions, but the mongodb one was experimental.
If they are now always the same, I'll change it when I write the guide (before 2.3)

Reply:
Cool, thanks. And yes, I just think it needs a change if we are guaranteed they are built at the same time. What I'm worried about is Dependabot updating one at a time and thus having partial failing PRs.

Reply:
So the version of the mongodb extension follow the one of the core liquibase but it's not release on the same day. I may have up to 10 days between a release of the core liquibase lib and the mongodb one.
There also can exist patch version for bugfixes on the liquibase mongodb extension (for ex 4.1.1.1 that don't exist for liquibase core).